src/main/java/com/algorand/algosdk/v2/client/model/Account.java

```diff
@@ -0,0 +1,119 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.v2.client.common.PathResponse;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Account information at a given round. 
+ * Definition: 
+ * data/basics/userBalance.go : AccountData 
+ * 
+ */
+public class Account extends PathResponse {
+
+	/**
+	 * the account public key 
+	 */ @JsonProperty("address")
```

Comment:
Looks like we still have a missing newline before `@JsonProperty`

Reply:
Also some weird indentation going on

Reply:
fixed

Reply:
fixed.